azure-pipelines.yml

```diff
@@ -25,8 +26,20 @@ jobs:
       vmImage: $(imageName)
     steps:
       - task: GoTool@0
+        env:
+          GO111MODULE: 'on'
         inputs:
           version: '1.16'
+
+      - script: |
+          curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.43.0
+          echo 'Installation complete' 
+          ./bin/golangci-lint --version 
+          ./bin/golangci-lint run e2etest
```

Comment:
is this basically the standard? since [golint](https://github.com/golang/lint) is deprecated.

Reply:
Have you considered adding `go vet` as well?

Reply:
Yes you're right. It's not the standard golint. It is the same one used by OneEngg team in Azure SDK Go Pipelines.

Reply:
As for the vet, I have not considered adding `go vet` but I can add it as long as it can be used for a particular directory.

Reply:
Ideally, we should have [these steps](https://github.com/Azure/azure-sdk-for-go/blob/main/eng/pipelines/templates/steps/analyze.yml) in CI as well.